opm/autodiff/BlackoilModelEbos.hpp

```diff
@@ -285,12 +281,11 @@ namespace Opm {
             std::vector<double> residual_norms;
             perfTimer.reset();
             perfTimer.start();
-            report.converged = getConvergence(timer, iteration,residual_norms);
+            // the step is not considered converged until at least minIter iterations is done
```

Comment:
If the residual is converged before any iterations, it will solve once (minIter = 1), updateState, invalidateCache, try to output (since report.converged is true) and fail.
With this change it will always linearize after the update, check  the residual again, if it is still converged it will output.
So  and alternative fix would be to call updateIntensiveQuantities if iter<minIter and report.converged. Do you want me to do that?